### DISCUSSION TITLE
Configuring unique vendorID, productID and device description on Raspberry Pi Pico

### DISCUSSION THREAD

**Discussion by 0i9j** [hathach/tinyusb#1649] [Q&A]
I want to configure the Rasperry Pi Pico with a unique Vendor, Product ID and device description. My issue right now is, that as soon as I implement the tinyUSB library, the Pico doesn´t show up as a ComPort anymore. I digged through the library files and the examples and figured that I can probably do my settings with the `tusb_desc_device_t const desc_device ={//add settings//}`. For now i would be happy if the Pico would show up as usual so as Serial Device (COM X) on my device manager on windows. I´d apprechiate any help, also if you need any more details, please let me know.

**Comment by hathach** [comment]
> that as soon as I implement the tinyUSB library

Can you specify exactly what did you do. The vid, pid can simply changed by updating the descriptor e.g  https://github.com/hathach/tinyusb/blob/master/examples/device/cdc_msc/src/usb_descriptors.c
